### PR TITLE
Deprecate inventory-web

### PR DESCRIPTION
**Pull request by jbrown-xentity**
Move to inventory-next for deployment, remove new relic deployment (handled as part inventory deployment)

### CODE REVIEW

File: ansible/inventory.yml [hunk 1]
```diff
@@ -65,19 +65,8 @@
             - rotate 8
 
 
-- name: NewRelic
-  hosts: inventory-web:!inventory-next
-  vars:
-    newrelic_app_name: inventory
-  roles:
-    - role: monitoring/newrelic/python-agent-ansible
```
Comment (adborden) [inline ansible/inventory.yml:73]: Can you remove this role? It's not used anymore with gunicorn.

**Comment by FuhuXia** [comment]
@jbrown-xentity Please verify `inventory_ckan_service_url` values in sandbox and staging is what we expect them to be.

**Comment by jbrown-xentity** [comment]
ping @adborden for re-review.

**Review by adborden — APPROVED** [review]
🚀

**Pull request merged**